db/migrate/20190301173059_create_pg_search_documents.rb

```diff
@@ -0,0 +1,82 @@
+class CreatePgSearchDocuments < ActiveRecord::Migration[5.2]
+  def self.up
+    say_with_time("Creating table for pg_search multisearch") do
+      create_table :pg_search_documents do |t|
+        t.text :content
+        t.tsvector :tsv_content
+        t.belongs_to :searchable, :polymorphic => true, :index => true
+        t.timestamps null: false
+      end
+    end
+    add_index(:pg_search_documents, :tsv_content, using: 'gin')
+
+    say_with_time("Adding trigger function on documents for updating tsv_content column") do
+
+      sql = <<MIGRATION
```

Comment:
Asi nechapem na co je ten trigger dobry. Ved ked sa updatne tsv_content tak sa proste aj index zmeni nie?

Reply:
[https://github.com/Casecommons/pg_search/wiki/Building-indexes](https://github.com/Casecommons/pg_search/wiki/Building-indexes)

ono ten pg_search gem updatuje stlpec `content` a ten trigger ma nasledne updatnut `tsv_content`

Reply:
Ok rozumiem, aj ked nechapem preco to takto divne vyrabali ked pg ma normalne function indexy kde to mohli mat "zadarmo" bez navyse stlpca.